tests/integration/targets/test_manage_transit_gateway/defaults/main.yml

```
@@ -0,0 +1,12 @@
+cidr_prefix: '10.{{ 255 | random(seed=resource_prefix) }}'
```

```suggestion
aws_security_token: '{{ security_token | default(omit) }}'
cidr_prefix: '10.{{ 255 | random(seed=resource_prefix) }}'
```

This is needed for `aws_setup_credentials` role, otherwise the security token will be taken into account